src/pages/registration/FileList.tsx

```diff
@@ -203,10 +202,6 @@ export const FileList = ({ title, files, uppy, remove, baseFieldName }: FileList
                   disabled={!userCanEditFile(file, user, values)}
                   removeFile={() => {
                     const associatedArtifactsBeforeRemoval = associatedArtifacts.length;
-                    const remainingFiles = uppy
-                      .getFiles()
-                      .filter((uppyFile) => uppyFile.response?.uploadURL !== file.identifier);
-                    uppy.setState({ files: remainingFiles });
```

Comment:
Mener vi hadde dette for å håndtere feil når man fjerner og legger til samme fil igjen, men klarer ikke se noen grunn til å ta vare på det nå. Foreslår at vi tar det vekk inntil noen ser en grunn til å ha det